src/utils/studyUtils.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Formats access types based on controlled and open access flags.
+ *
+ * @param {boolean} controlledAccess - Indicates whether controlled access is enabled.
+ * @param {boolean} openAccess - Indicates whether open access is enabled.
+ * @returns A formatted string listing the enabled access types separated by a comma and space.
+ *          - Returns "Controlled, Open" if both are enabled.
+ *          - Returns "Controlled" if only controlled access is enabled.
+ *          - Returns "Open" if only open access is enabled.
+ *          - Returns an empty string if neither is enabled.
+ */
+export const formatAccessTypes = (controlledAccess: boolean, openAccess: boolean) => {
```

Comment:
Based on the latest reqs, should this be "All" instead of "Controlled, Open"? Maybe that requirement was accidentally updated.

Also, if we're updating this function, it probably doesn't hurt to type the response too. Otherwise, disregard this.


```suggestion
export const formatAccessTypes = (controlledAccess: boolean, openAccess: boolean): string => {
```